src/resolvers/topicResolvers.ts

```diff
@@ -132,7 +132,10 @@ export const resolvers: { Topic: GQLTopicTypeResolver<TopicResponse> } = {
         { id: topic.id, filterIds: args.filterIds },
         context,
       );
-      return filterMissingArticles(subtopics, context);
+      const visibleTopics = subtopics.filter(sub =>
+        sub.metadata ? sub.metadata.visible : true,
+      );
+      return filterMissingArticles(visibleTopics, context);
```

Comment:
For meg ser det ut som `filterMissingArticles` allerede gjør denne filtreringen?

Reply:
Det er jammen sant. Eg var sikker på at eg fikk opp et usynlig emne i test og spora det tilbake hit. Kanskje koden ikkje har slått inn tidligere, men no returnerers metadata for alle endepunkt uten at vi må be om det.